src/gateway/environments/jsdom-sixteen/__tests__/__snapshots__/jsdom-sixteen-environment.test.js.snap

```diff
@@ -1,13 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`JSDOMSixteenEnvironment #render should reject if result is malformed 1`] = `"Malformed render result: undefined"`;
+exports[`JSDOMSixteenEnvironment #render should reject if result is malformed ({ body: 'NEED MORE THAN THIS' }) 1`] = `"Malformed render result: {\\"body\\":\\"NEED MORE THAN THIS\\"}"`;
```

Comment:
I made it so the test case included the data point so it was easier to map from here to the test values. In reality, the snapshots didn't change.